src/Fable.Transforms/Rust/AST/Rust.AST.Symbols.fs

```diff
@@ -167,7 +167,7 @@ module kw =
     let RustPrelude = [
         "Copy"; "Send"; "Sized"; "Sync"; "Unpin";
         "drop"; "Drop"; "Fn"; "FnMut"; "FnOnce";
-        "Box"; "ToOwned"; "Clone";
+        "ToOwned"; "Clone";
```

Comment:
@alexswan10k Please undo this change, we can't change the fact what's in the Rust prelude.
If you need to avoid it when creating a symbol, you can use `rawIdent` (see below).